snmp_base_block.py

```diff
@@ -33,7 +33,7 @@ class SNMPBase(LimitLock, Block):
     """
 
     agent_host = ExpressionProperty(title="SNMP Agent Url", default='127.0.0.1')
-    agent_port = IntProperty(title="SNMP Agent Port", default=161)
+    agent_port = ExpressionProperty(title="SNMP Agent Port", default='161')
```

Comment:
I think it should be ok. Technically the `ExpressionProperty` inherits from the `StringProperty` so it should call `str()` on any input first, which should turn our `int` in config files into a string